silverback/_cli.py

```diff
@@ -26,6 +27,20 @@
 from silverback.runner import PollingRunner, WebsocketRunner
 from silverback.worker import run_worker
 
+from .exceptions import ImportFromStringError
+
+DOCKERFILE_CONTENT = """
+FROM ghcr.io/apeworx/silverback:latest
+USER root
+WORKDIR /app
+RUN mkdir -p ./bots && chown harambe:harambe /app/bots
```

Comment:
Also copy `bot*` (includes `bot.py` but also if there's a `bot/` module)

Reply:
I'm just creating a bots/ folder here. In the `generate-dockerfile` method, I add a line to the dockerfile to `COPY` the file into the `bots/` folder to run. I'm treating the bots folder as a scripts folder. I need some clarity on what you're expecting the folder structure to look like